rdtho2525/Niche_GoT
# Feature/e2e testing

## Is this a fix or a feature?
  + fix

## What is the change?
  + Conducts end-to-end testing
  + Replaces broken API call with local mocked data

## Where should the reviewer start?
  + cypress/integration

## How should this be tested?
  + All functionality should be maintained
  + Run command `npm run cypress:open`
  + All but the 2 tests regarding network requests should be passing
